game_rules.py

```diff
@@ -0,0 +1,27 @@
+class GameRules:
+    def __init__(self):
+        pass
+
+    def horizontal_win(self, board):
+        if board.spots[0] == board.spots[1] == board.spots[2] or board.spots[3] == board.spots[4] == board.spots[5] or board.spots[6] == board.spots[7] == board.spots[8]:
+            return True
```

Comment:
Similar to line 18 in win method, you can refactor every method in this class to return without using an if-statement.

Reply:
fixed this and couple more alike in https://github.com/olgaboiar/ttt_python/pull/6/commits/08045efaf06c0c7c241b9c556250498a2ec2627f